lib/lndclient/LndClient.ts

```diff
@@ -53,7 +53,7 @@ class LndClient extends BaseClient {
   private credentials!: ChannelCredentials;
   private reconnectionTimer?: NodeJS.Timer;
   private identityPubKey?: string;
-
```

Comment:
Here too I'd prefer to keep the blank line separating `private` from `private static` fields.